src/modules/recursive_partitioning/decision_tree.cpp

```diff
@@ -235,7 +259,14 @@ dt_apply::run(AnyType & args){
     else{
         return_code = TERMINATED;  // indicates termination due to error
     }
+    uint32_t n_valid_nodes = 0;
+    for (Index i=0; i < dt.feature_indices.size(); i++){
+        if (dt.feature_indices(i) != dt.NODE_NON_EXISTING)
+            n_valid_nodes ++;
+    }
 
+    elog(INFO, "Depth = %d, n_valid_nodes = %d",
```

Comment:
This info and its context (the sql function that calls madlib._dt_apply) is printed to the console if the user is calling the DT from a plain psql client.